src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java

```diff
@@ -268,16 +267,14 @@ public void testCacheDirectoryDoesNotExistAndShouldBeCreated() throws IOExceptio
     @Test
     public void testPathToCacheContainsOnlyFileName() throws IOException {
         final Configuration config = new DefaultConfiguration("myName");
-        final String fileName = "temp.cache";
-        final Path filePath = Paths.get(fileName);
-        final PropertyCacheFile cache = new PropertyCacheFile(config, fileName);
+        final String filePath = File.createTempFile("junit", null, temporaryFolder).getPath();
```

Comment:
`getPath` => `getName`

Reply:
Why?
We use to have path.

Reply:
`testPathToCacheContainsOnlyFileName `

Reply:
hmm, in this case it creates junit15239200722429684874.tmp

and after digging deeper, this test failed becuse for some reason my local I got this file as folder, but it suppose to be a file,
weird, .... I am closing PR as no change should be done, and i do not know how to reproduce problem with folder creation instead of file